backend/dal/Services/Concrete/AdminParcelService.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Entities;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+
+namespace Pims.Dal.Services
+{
+    //TODO: Comments
```

Comment:
Not worrying about polish until I get confirmation that I'm going in the right direction.

Reply:
I think for our purposes at this point, this looks good.  Just make sure the DI works with multi-threading.  I think we probably are okay.  Note that if a user hasn't logged in the ClaimsPrincipal will be null.  Not sure if DI will have a copy at that time or not, or how it will be handled.  It could mean a call here would cause a failure.  Which may not in itself be an issue, as we only wanted authenticated users using this service, but the error may not be clear.

Reply:
I'll see about doing some of that testing, I still need to run through the postman apis to validate everything is working, just wanted to make sure I wasn't going down the wrong path.